pkg/postgres/pgutils/utils.go

```diff
@@ -84,6 +100,19 @@ func EmptyOrMap[K comparable, V any, M map[K]V](m M) interface{} {
 
 // CreateTableFromModel executes input create statement using the input connection.
 func CreateTableFromModel(ctx context.Context, db *gorm.DB, createStmt *postgres.CreateStmts) {
+	var rawSQL *gorm.DB
```

Comment:
It looks like this new variable is not used in every location where the `db` function parameter was used. Is it intentional ?
Possible missed calls are
- line 120, in the non-partition block
- line 130, when recursively applying the changes to the children tables

Reply:
I'm not a huge fan of the name `rawSQL` as to me it implies actual SQL when in fact it is just another instance of `gorm.DB`.  That said, naming is hard and I don't really have any better suggestions unless we go `rawSQLDB`